src/components/ViolenceTestCards/index.jsx

```diff
@@ -26,7 +26,7 @@ const ViolenceTestCards = () => {
 
   const cards = cardsArr.map((card) => {
     return (
-      <div className="col">
+      <div className="col" key={card.id}>
```

Comment:
you need a key everywhere that you use map in react, including the other spot. if i was merely saying ID because your old code used to use ID, but if i were to do it, i would just assume that all category titles and checkbox items are unique

`key={card.category}`

and `key={item}` (for checkbox items)

Reply:
adding an id to card is a bit cumbersome to maintain

Reply:
ok I added the keys